doc/subsystems/networking/net-app-api.rst

```diff
@@ -5,17 +5,23 @@ Network Application API
 
 The Network Application (net-app) API allows applications to:
 
-* **Initialize** the application for networking use. This means, for example,
+**Initialize**
+
```

Comment:
For this to show up as a definition list, this blank line needs to be removed between the term and it's definition.  (Otherwise the "definition" shows up as a block quote.)